File: C/2020/12限/inc_dec.c
//  inc_dec.c
//  cl  inc_dec.c
#include <stdio.h>

int main() {
    int i = 0;
    i ++;
    ++ i;
    printf("%d \n", i);
    i --;
    -- i;
    printf("%d \n", i);
    //
    printf("%d \n", i ++); // 0
    printf("%d \n", ++ i); // 2

    return 0;
}

File: Php/2025/EarlyPeriodTue34/incdec.php
<?php  // incdec.php

$i = 0;
echo "i is {$i} <br>";

$i = $i + 1;
echo "i is {$i} <br>";

$i += 1;
echo "i is {$i} <br>";

++ $i; // 加算子(インクリメント): 前置
echo "i is {$i} <br>";

$i ++; // 加算子(インクリメント): 後置
echo "i is {$i} <br>";

//
-- $i; // 減算子(デクリメント): 前置
echo "i is {$i} <br>";

$i --; // 減算子(デクリメント): 後置
echo "i is {$i} <br>";

$i -= 1;
echo "i is {$i} <br>";
